sdk/vaa/payloads.go

```diff
@@ -279,6 +296,23 @@ func (r *BodyWormchainWasmAllowlistInstantiate) Deserialize(bz []byte) {
 	r.CodeId = codeId
 }
 
+func (r BodyWormchainIbcComposabilityMwContract) Serialize() []byte {
+	payload := &bytes.Buffer{}
+	payload.Write(r.ContractAddr[:])
+	return serializeBridgeGovernanceVaa(WasmdModuleStr, ActionSetIbcComposabilityMwContract, ChainIDWormchain, payload.Bytes())
```

Comment:
Is `WasmdModule` appropriate for the IBC composability middleware contract? Does it have to do with wasmd at all?

Reply:
Moved the governance action to the new `GatewayModule`